ehr/resources/schemas/dbscripts/sqlserver/ehr_lookups-22.000-22.001.sql

```diff
@@ -0,0 +1,20 @@
+ALTER TABLE ehr_lookups.cage ADD Lsid LSIDtype;
+ALTER TABLE ehr_lookups.areas ADD Lsid LSIDtype;
+
+CREATE TABLE ehr_lookups.floors (
+    rowId int identity(1,1),
+    floor varchar(100),
+    building varchar(100),
+    name varchar(100),
+    description varchar(100),
```

Comment:
Is it possible for them to have non-ascii characters? better to use nvarchar on sql server.

Reply:
Good call, updated to NVARCHAR